src/db/spanner/batch.rs

```diff
@@ -355,23 +355,28 @@ pub async fn do_append_async(
         } else {
             let sortindex = bso
                 .sortindex
-                .map(|sortindex| as_value(sortindex.to_string()))
+                .map(|sortindex| sortindex.to_spanner_value())
```

Comment:
Why not use the same `ToSpannerValue::to_spanner_value` method you do below?

Reply:
An excellent point!